GDJS/Runtime/runtimeobject.js

```diff
@@ -88,6 +88,10 @@ gdjs.RuntimeObject = function(runtimeScene, objectData)
         this._behaviorsTable.put(autoData.name, this._behaviors[i]);
     }
     this._behaviors.length = i;//Make sure to delete already existing behaviors which are not used anymore.
+
+    //Timers:
+    if (this._timers === undefined)
+        this._timers = new Hashtable();
```

Comment:
```
else {
  this._timers.clear();
```

otherwise a object could be reused/recycled from another and get the same timers :D